src/Language/PureScript/Sugar/ObjectWildcards.hs

```diff
@@ -21,15 +22,14 @@ desugarObjectConstructors
   => Module
   -> m Module
 desugarObjectConstructors (Module ss coms mn ds exts) = Module ss coms mn <$> mapM desugarDecl ds <*> pure exts
-  where
 
-  desugarDecl :: Declaration -> m Declaration
-  desugarDecl (PositionedDeclaration pos com d) = rethrowWithPosition pos $ PositionedDeclaration pos com <$> desugarDecl d
-  desugarDecl other = f other
-    where
-    (f, _, _) = everywhereOnValuesTopDownM return desugarExpr return
+desugarDecl :: (MonadSupply m, MonadError MultipleErrors m) => Declaration -> m Declaration
```

Comment:
Actually, this code works with more than just objects now, so maybe these functions (and the module) should all be renamed to just `...Wildcards`.